library/Fission/Models.hs

```diff
@@ -60,26 +75,160 @@ User
 
   deriving Show Eq
 
-UserCid
-  userFk     UserId
+------------
+-- Events --
```

Comment:
Events are just normal SQL records right? But they're used as an append-only log of all of the transformations?

Reply:
@dholms yep!

* Regular SQL table
* Append-only = no delete or update operations